scrape_all.sh

```diff
@@ -0,0 +1,2 @@
+scrapy crawl brandenburg -s "CLOSESPIDER_ITEMCOUNT=3"
+scrapy crawl niedersachsen -s "CLOSESPIDER_ITEMCOUNT=3"
```

Comment:
You only included Brandenburg and Niedersachsen. Is there anything that keeps us from expanding this to the other states?
Also, what are you trying to achieve with `-s "CLOSESPIDER_ITEMCOUNT=3"`?

Reply:
Fixed in 3bb2795